cfgov/v1/forms.py

```diff
@@ -70,22 +70,23 @@ def __init__(self, *args, **kwargs):
 
 
 class FilterableListForm(forms.Form):
-    title_attrs = {
-        'placeholder': 'Search for a specific word in item title'
-    }
     topics_select_attrs = {
         'multiple': 'multiple',
         'data-placeholder': 'Search for topics',
+        'class': 'o-multiselect',
+        'id': 'topics'
     }
     authors_select_attrs = {
         'multiple': 'multiple',
-        'data-placeholder': 'Search for authors'
+        'data-placeholder': 'Search for authors',
+        'class': 'o-multiselect',
+        'id': 'authors'
     }
 
     title = forms.CharField(
         max_length=250,
```

Comment:
I think it can be, or, if we wanted, we could choose to go the other way and move more of the widget styling into Python. As of Django 1.11 we could consider making Capital Framework-compatible widgets, e.g. `widget=widgets.CapitalFrameworkTextInput` that would encapsulate this styling and make it easier to reproduce across the site. It'd also automatically handle things like the missing `value` attribute mentioned above, as Django would render the widget instead of us manually doing it. This came up before [with the conference registration form](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/data_research/forms.py#L10). That might be a larger discussion/design decision, though, and this is a good solution for now.